### PR TITLE
Add rokuhentai

### PR DESCRIPTION
**Pull request by cc713**
The htdoujin extractor issue seems to be caused by `downloader.go`, use 1.2.31 version works for me.

### CODE REVIEW

**Comment by gan-of-culture** [comment]
the rokuhentai extractor test failed. Please take a look at it^^ As for the https://hentaitv.fun/ extractor it's like you said the videos don't load on the website right now, which makes the hentaitv extractor fail. Don't worry about that for now 👍

**Comment by cc713** [comment]
I can't find what's wrong. Local test returns `ok`.
Below is what github checks returns
```
--- FAIL: TestExtract (0.26s)
    --- FAIL: TestExtract/Single_Gallery (0.26s)
panic: runtime error: index out of range [1] with length 0 [recovered]
	panic: runtime error: index out of range [1] with length 0

goroutine 25 [running]:
testing.tRunner.func1.2({0x6ce640, 0xc0000e21e0})
	/opt/hostedtoolcache/go/1.19.2/x64/src/testing/testing.go:1396 +0x24e
testing.tRunner.func1()
	/opt/hostedtoolcache/go/1.19.2/x64/src/testing/testing.go:1399 +0x39f
panic({0x6ce640, 0xc0000e21e0})
	/opt/hostedtoolcache/go/1.19.2/x64/src/runtime/panic.go:884 +0x212
github.com/gan-of-culture/get-sauce/extractors/rokuhentai.extractData({0x6f09e3, 0x1d})
	/home/runner/work/get-sauce/get-sauce/extractors/rokuhentai/rokuhentai.go:73 +0x4c5
github.com/gan-of-culture/get-sauce/extractors/rokuhentai.(*extractor).Extract(0xf?, {0x6f09e3?, 0x39?})
	/home/runner/work/get-sauce/get-sauce/extractors/rokuhentai/rokuhentai.go:35 +0xc5
github.com/gan-of-culture/get-sauce/extractors/rokuhentai.TestExtract.func1(0xc000085ba0)
	/home/runner/work/get-sauce/get-sauce/extractors/rokuhentai/rokuhentai_test.go:52 +0x3b
testing.tRunner(0xc000085ba0, 0xc000082ef0)
	/opt/hostedtoolcache/go/1.19.2/x64/src/testing/testing.go:1446 +0x10b
created by testing.(*T).Run
	/opt/hostedtoolcache/go/1.19.2/x64/src/testing/testing.go:1493 +0x35f
FAIL	github.com/gan-of-culture/get-sauce/extractors/rokuhentai	0.265s
```

File: extractors/rokuhentai/rokuhentai.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+func extractData(URL string) (*static.Data, error) {
+	htmlString, err := request.Get(URL)
+	if err != nil {
+		return nil, err
+	}
+
+	ID := utils.GetLastItemString(reID.FindStringSubmatch(URL))
+
+	matchedPageInfo := rePageInfo.FindStringSubmatch(htmlString)
```
Comment (gan-of-culture) [inline extractors/rokuhentai/rokuhentai.go:72]: Try adding a IF statement here to check the length of `matchedPageInfo`. If < than what you expect you can print the `htmlString` using `fmt.Println(htmlString)` and then `return`.

The printed output should be visible in the job log then.

**Comment by gan-of-culture** [comment]
Closed due to inactivity. Feel free to reopen it at a later point.

**Pull request closed without merging**